tools/copy_production_db_to_staging

```diff
@@ -0,0 +1,162 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${OAR_DEBUG}" ]]; then
+    set -x
+fi
+
+# Constants
+STAGING_ID="openapparelregistry-enc-stg"
+PRODUCTION_ID="openapparelregistry-enc-prd"
+ALLOWED_EMAILS="person@example.com other@example.com"
```

Comment:
@jwalgran do we have the actual list we want here? Or do we not want to list the default like with the password fields?